src/pages/worklist/MessageAccordion.tsx

```diff
@@ -55,57 +55,46 @@ const StyledAccordionActionButtons = styled.div`
   justify-content: flex-end;
 `;
 
-interface DoiRequestAccordionProps {
-  identifier: string;
+interface MessageAccordionProps {
+  message: Message;
+  refetchMessages: () => void;
 }
 
-export const DoiRequestAccordion = ({ identifier }: DoiRequestAccordionProps) => {
+export const MessageAccordion = ({ message, refetchMessages }: MessageAccordionProps) => {
   const { t } = useTranslation('workLists');
   const dispatch = useDispatch();
-  const [isSendingMessage, setIsSendingMessage] = useState(false);
-  const [registration, , refetchRegistration] = useFetchRegistration(identifier);
-
-  if (!registration?.doiRequest) {
-    return null;
-  }
+  const identifier = message.publication.identifier;
 
   const onClickSendMessage = async (message: string) => {
-    setIsSendingMessage(true);
-    const updatedDoiRequestWithMessage = await updateDoiRequestWithMessage(identifier, message);
+    const updatedDoiRequestWithMessage = await addMessage(identifier, message);
     if (updatedDoiRequestWithMessage) {
       if (updatedDoiRequestWithMessage.error) {
-        dispatch(setNotification(t('feedback:error.message_failed'), NotificationVariant.Error));
+        dispatch(setNotification(t('feedback:error.send_message'), NotificationVariant.Error));
       } else {
-        dispatch(setNotification(t('feedback:success.message_sent'), NotificationVariant.Success));
-        refetchRegistration();
+        dispatch(setNotification(t('feedback:success.send_message'), NotificationVariant.Success));
+        refetchMessages();
       }
-      setIsSendingMessage(false);
     }
   };
 
   return (
     <StyledAccordion data-testid={`doi-request-${identifier}`}>
       <AccordionSummary expandIcon={<ExpandMoreIcon fontSize="large" />}>
         <StyledStatus data-testid={`status-doi-request-${identifier}`}>
-          {t(`doi_requests.status.${registration?.doiRequest.status}`)}
+          {message.messages.some((m) => m.isDoiRequestRelated) ? t('types.doi') : t('types.support')}
         </StyledStatus>
         <StyledTitle data-testid={`title-doi-request-${identifier}`}>
-          {registration?.entityDescription?.mainTitle}
+          {message.publication.entityDescription?.mainTitle}
         </StyledTitle>
         <StyledOwner data-testid={`owner-doi-request-${identifier}`}>
-          <Label>{registration?.owner}</Label>
-          {new Date(registration?.doiRequest.createdDate).toLocaleDateString()}
+          <Label>{message.publication.owner}</Label>
+          {new Date(message.messages[message.messages.length - 1].date).toLocaleDateString()}
```

Comment:
`message.messages` høres veldig rart ut 😅  hva betyr dette egentlig? er det riktig at parent skal hete `message`? er `message`=`conversation` her eller?

Reply:
kan være det blir endringer på dette ganske snart. har sendt dette forslaget til backend:
```
{
  "publication": {...},
  "conversations": [
      { 
         "type": "Support",
         "status": "Resolved",
         "messages": [...]
      },
      { 
         "type": "DoiRequest",
         "status": "Pending",
         "messages": [...]
      }
   ]
}
```

Reply:
`Chats` kunne kanskje også funket?

Reply:
Orestis skal snart se på dette, så det kommer sikkert endringer ene eller andre veien snart. lar det ligge til da

Reply:
I mellomtiden så kan du jo kose deg med å oppdatere scenarioet slik at det faktisk gjenspeiler sånn som det blir med denne PRen? 

https://github.com/BIBSYSDEV/NVA-scenarios/blob/master/1237-registrator_worklist/1238-creator_opens_my_messages.feature

Reply:
nja, vil vi egentlig det? dette er ikke noen ferdig versjon, og figma-skissene ser helt annerledes ut (men heller ikke de er spikret som jeg vet). hvis vi endrer scenario til å stemme med denne PRen må vi ta bort ting som må legges til igjen senere, så kan like greit vente til skissene er ferdige imo

Reply:
men hvordan vet du at det du har gjort nå er "riktig" da? Jeg synes vi bør være ganske strenge med å få på plass features før vi begynner utviklingen. Kan ende opp i mye dobbeltarbeid hvis vi ikke krever feature først...

Reply:
Kan legge til `Support` som ny type (kanskje det du mente også), ser det har kommet med i scenario for Worklist men ikke My Messages. Scenario sier nå at vi skal vise status også, men det har vi ikke støtte for enda.

så får vi en dag en jobb med å oppdatere design for både worklist og my messages:
![bilde](https://user-images.githubusercontent.com/6313468/110796503-4f1e3c00-8278-11eb-9eac-c82969415a0f.png)
![bilde](https://user-images.githubusercontent.com/6313468/110796605-6eb56480-8278-11eb-86f6-777b05d987b2.png)

Reply:
ja, var support som ny type som jeg mente hovedsaklig ja 👍

Reply:
skulle det renames i backend?

Reply:
jepp, kommer noen endringer her når Orestis får ut siste i dev. Samme for meg om vi tar det med i denne eller lagen egen PR på det etterpå

Reply:
kanskje vi bare skal få merget denne nå allikevel da? så blir det ikke så stor PR